ecs-init/docker/docker_test.go

```diff
@@ -253,6 +253,14 @@ func validateCommonCreateContainerOptions(opts godocker.CreateContainerOptions,
 	if hostCfg.NetworkMode != networkMode {
 		t.Errorf("Expected network mode to be %s, got %s", networkMode, hostCfg.NetworkMode)
 	}
+
+	if len(hostCfg.CapAdd) != 1 {
```

Comment:
Ah, alright then. I think we should vendor that for this package. May be not for this PR, but it'd be nice to start using some `assert`'s here.